app/uk/gov/hmrc/hec/models/hecTaxCheck/SAStatus.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.hec.models
+package uk.gov.hmrc.hec.models.hecTaxCheck
```

Comment:
how do we determine that something like this goes into `hecTaxCheck` but something like `CorrectiveAction` or `TaxSituation` doesn't?

Reply:
my thinking was anything that's used in `HECTaxCheckData` goes into `hecTaxCheck`

Reply:
I think then that maybe everything that's remaining except for `Error` and `TaxCheckListItem` would probably go into `hecTaxCheck` cos they're only being used in the stored data structure

Reply:
then does it make sense to have a separate package for it? or would be just as nice under `models` do you think?

Reply:
having a separate package seems to make it neater I think - and then it kind of makes sense cos `Error` is something which is global to the service. Not sure about `TaxCheckListItem` though cos that's quite specific...maybe that kind of belongs in `hecTaxCheck` as well cos it's to do with tax check data

Reply:
updated https://github.com/hmrc/hec/pull/54/commits/aa47933b2d812011e367018d0a077f329c4e5a24

I've left `TaxCheckListItem` under `models` since it's not stored in the db